leaflet/leaflet.d.ts

```diff
@@ -805,7 +805,7 @@ declare namespace L {
          * }
          * ```
          */
-        pointToLayer?: (geoJsonPoint: GeoJSON.Point, latlng: LatLng) => Layer; // should import GeoJSON typings
+         pointToLayer?: (geoJsonPoint: GeoJSON.Point|GeoJSON.Feature<GeoJSON.Point>, latlng: LatLng) => Layer; // should import GeoJSON typings
```

Comment:
I tried to have a GeoJSON.Point here...   It is is always wrapped with a feature.
Even when only a geometrycollection is given, a new feature is generated by leaflet.
http://jsfiddle.net/JanDeclercq/sk539uds/6/

http://leafletjs.com/reference.html#geojson-pointtolayer  
the parameter name in the documentation also says:  featureData.      This could also be a hint that it is always a feature.

All the examples from leaflet itself are written like this:
pointToLayer: function (feature, latlng)    another hint in the parameter name about it  always beeing a feature.

I believe it is save to change this to GeoJSON.Feature<GeoJSON.point>